### PR TITLE
Sepa overlay view dynamic content

### PR DESCRIPTION
**Pull request by mobileapps23**
1. changed appearance of Submit button 
2. changed view model for SepaOverlayView - removed Submit button colors
3. changed layout for SepaOverlayView - content wrapped to scrollView, and it will allow to display labels with any string length (as "text"label coming from projects settings) and content size category
4. changed configure() method - for correct scrollView positioning it should now its frame height.

Steps to check:
Open project Pflanzen, choose  Sepa for payment
Enter test credentials for Sepa: DE89 3704 0044 0532 0130 00
Put something to cart, press Jetzt bezahlen

### CODE REVIEW

**Review by mrgrauel — APPROVED** [review]
I changed the logic see e100a3352d41e1c7ab610c7925891b689b14efd9. The system calculates everything. No need for some manual calculations or do you find an edge case?

File: Snabble/UI/Payment/PaymentMethod+Start.swift [hunk 1]
```diff
@@ -58,6 +59,7 @@ public final class PaymentMethodStartCheck {
 
     // MARK: - Sepa
     @objc private func dismissOverlay(_ sender: Any) {
+        print("close tapped")
```
[the remark this answers is not in the document]
Comment (mrgrauel) [reply Snabble/UI/Payment/PaymentMethod+Start.swift]: 7a2bc864ddeba15c9225fe4c3b46f00b85fa04a6

**Comment by mobileapps23** [comment]
If text string has more length (and as it's from Project settings, it could be any length), then it will be trunkated, not scrolling
1. tap to checkout, that SepaOverlaView appeares on screen
2. increase dynamic font category
3. Text not scrollable

image

**Comment by mrgrauel** [comment]
As you can see here, it's working for me. Do you have the head on 7a2bc864ddeba15c9225fe4c3b46f00b85fa04a6?

https://user-images.githubusercontent.com/148758/171614339-0c2ef257-6ce2-48ab-b25d-60effa715bf0.MP4

**Comment by mrgrauel** [comment]
Ahh I See what you mean, as soon as you change the font when you are already on the screen 🤔

**Comment by mrgrauel** [comment]
@mobileapps23 fixed with 99217b9b6e619cc1fb5d882654eb267d5a2b4b64.

**Pull request merged**